### PR TITLE
[NFC][SYCLomatic][DOC]Add manually fixed case about L2cache/SLM API for 1026/1027 warning msg

### PR DESCRIPTION
**Pull request by wenhuiNi**
Signed-off-by: Ni, Wenhui <wenhui.ni@intel.com>

### CODE REVIEW

**Review by zhiweij1 — APPROVED** [review]

File: docs/dev_guide/diagnostic_ref/dpct1026.rst [hunk 1]
```diff
@@ -74,41 +91,72 @@ results in the following migrated SYCL code:
...
+     return;
+   }
    
    void foo() {
      cudaLimit limit;
      dpct::queue_ptr stream;
      unsigned int flags;
      int config;
-   
+     float *aptr;
```
Comment (zhiweij1) [inline docs/dev_guide/diagnostic_ref/dpct1026.rst]:
```suggestion

```

File: docs/dev_guide/diagnostic_ref/dpct1026.rst [hunk 2]
```diff
@@ -74,41 +91,72 @@ results in the following migrated SYCL code:
...
-     DPCT1026:6: The call to cudaSetDeviceFlags was removed because SYCL currently
+     DPCT1026:7: The call to cudaSetDeviceFlags was removed because SYCL currently
      does not support setting flags for devices.
      */
      /*
-     DPCT1026:7: The call to cuInit was removed because this call is redundant in
+     DPCT1026:8: The call to cuInit was removed because this call is redundant in
      SYCL.
      */
+     /*
```
Comment (zhiweij1) [inline docs/dev_guide/diagnostic_ref/dpct1026.rst:138]:
```suggestion
     float *aptr;
     /*
```

File: docs/dev_guide/diagnostic_ref/dpct1026.rst (hunk 1 above)
Comment (zhiweij1) [reply docs/dev_guide/diagnostic_ref/dpct1026.rst]: The location of the declartion of aptr is different from the CUDA code.

File: docs/dev_guide/diagnostic_ref/dpct1027.rst [hunk 3]
```diff
@@ -110,6 +130,41 @@ results in the following migrated SYCL code:
      redundant in SYCL.
      */
      err = 0;
+     float *aptr;
```
Comment (zhiweij1) [inline docs/dev_guide/diagnostic_ref/dpct1027.rst:133]: Same issue as above.

**Review by zhiweij1 — APPROVED** [review]

**Review by ziranzha — APPROVED** [review]

**Review by bdmoore1 — COMMENTED** [review]
Reviewed, no suggestions.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**